doc/ref_model/chapters/chapter03.md

```diff
@@ -508,12 +509,74 @@ Figure 3-10 shows a simple architecture of an SFC with multiple VNFs, as SF data
 <p align ="center"><b>Figure 3-10:</b> SFC Architecture for VNF based SFs </p>
 
 
-Figure 3-11 shows a simple architecture of an SFC with multiple CNFs, as SF data plane components, along with SFC management and CNF MANO components. 
+Figure 3-11a shows a simple architecture of an SFC with multiple CNFs, as SF data plane components, along with SFC management and CNF MANO components. 
 <p align="center"> <img src="../figures/ch03-model-sfc-architecture-cnf-2.png" alt="SFC Architecture for CNF based SFs" Title="SFC Architecture for CNF based SFs" width="45%"/></p>
-<p align ="center"><b>Figure 3-11:</b> SFC Architecture for CNF based SFs</p>
+<p align ="center"><b>Figure 3-11a:</b> SFC Architecture for CNF based SFs</p>
 
 The SFC management components together with the control components are responsible for rendering SFC requests to Service Function paths. For this they convert requisite SFC policies into network topology dependent paths and forwarding steering policies. Relevant SFC data components - classifiers, service function forwarders - are responsible for managing the steering policies.
 
+
+<a name="3.5.5.3"></a>
+#### 3.5.5.3 Information Flows in Service Function Chaining
+ 
+<a name="3.5.5.3.1"></a>
+##### 3.5.5.3.1 Creation of Service Function Chain
+ 
+The creation of the SFC might include design/preparation phase as: 
+-	The service functions that are included in the SFC.
+- The routing order in the service function, if the SFC is composed of more than one service function.
+
+Figure 3-11b shows SFC creation call flow, separated logically in two steps.
+<p align="center"> <img src="../figures/ch03-model-sfc-info-create-flow.png" alt="Creation of Service Function Chain" Title="Creation of Service Function Chain" width="45%"/></p>
+<p align ="center"><b>Figure 3-11b:</b> Creation of Service Function Chain</p>
+
+1.	Creation of service functions of SFC.
+
+The flow of steps to enable the SFC creation can be as follows:-
+
+a.	SFC orchestrator creates the SFs with help of VNF MANO or CNF MANO.
```

Comment:
@CsatariGergely i illustrated the fllow common  , CNF mano means devops related with CNF and you are right kubernetes does some of functionality.  i can rename picture as cnf devops rather than MANO . 
SFC orchestrator as such not specific open source i am aware , but ONAP dublin release have once module to take care of same - https://wiki.onap.org/pages/viewpage.action?pageId=45310767 
OpenDaylight supports SFC functionality needed for sfc relaization. Openstack neutron supports the same.
Hope i am able to answer you or you are looking something specific?